habitat-hitl/habitat_hitl/_internal/hitl_driver.py

```diff
@@ -10,12 +10,14 @@
 
 import abc
 import json
+import logging
 from datetime import datetime
 from functools import wraps
 from typing import TYPE_CHECKING, Any, Callable, Dict, List, Optional
 
 import magnum as mn
 import numpy as np
+from habitat_llm.utils.analysis import CodeTimer
```

Comment:
We'll want to wrap this in a try/except, otherwise HITL examples will fail for all users. A utility class would be good to wrap this.

Reply:
Indeed, example tests are failing. Should be as simple as checking for the dependency.